helm/chart-operator/templates/priorityclass.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if .Values.e2e  }}
+{{- if or .Values.chartOperator.cni.install .Values.e2e  }}
```

Comment:
The priority class is created via ignition but that won't work for CAPI clusters.